app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>User Detials "Index Page"</h1>
\ No newline at end of file
+<h1>User Detials "Show Page"</h1>
+<div>
+<h2>User</h2>
+  <img src="#{<%= @user.photo %>}" alt="User Img" />
+  <span><%= @user.name %></span>
+  <span><%= @user.posts_counter %></span>
+</div>
+<div>
+<h2>Bio</h2>
+  <p> <%= @user.bio || "the bio is embty" %></p>
+</div>
+<div>
+<h2>Posts</h2>
+<ol>
+<% @posts.each do |post| %>
```

Comment:
- [ ] Consider in the show view to show the recent 3 posts, not all posts by creating a method for the recent 3 posts.

Reply:
Implemented 👍🏻